content/posts/a_confusing_build_failure.md

```diff
@@ -9,56 +9,50 @@ tags: ["testing", "continuous integration"]
 
 Nowadays it has become quite a common practice to build and check software on machines other than local workstations, most of the time called CI servers(CI stands for [Continuous Integration](https://martinfowler.com/articles/continuousIntegration.html)). This can be done manually or automatically, usually triggered by a pull/merge request or a direct push on the main trunk(in git terminology most of the times named *master*).
 
-The point is that there's a single point where binaries are packaged and code is checked for quality and correctness, so we can merge changes safely and with confidence. 
+The upside is, there's a single point where binaries are packaged and code is checked for quality and correctness, so we can merge changes safely and with confidence. 
 
-When a failure appears in a build it should be investigated why did it happen. Did a commit actually break functionality/introduce a bug? Was static code analysis checks too harsh and maybe they must be relaxed a little bit? One of the failures that left me most perplexed was a build that was failing on the CI server unit tests but passed locally! How did this happen? Investigating on this confusing build failure unearthed bigger problems than just a breaking unit
-test.
+When a failure appears in a build it should be investigated, why did it happen? Did a commit break functionality/introduce a bug? Was static code analysis checks too harsh and maybe they must be relaxed a little bit? One of the failures that left me most perplexed was a build that was failing on the CI server unit tests but passed locally! How did this happen? Investigating on this confusing build failure unearthed bigger problems than just a breaking unit test.
 
 ## Mockito cannot mock final classes
 
-So why was the build failing? When the unit tests ran some of them would fail with an error stating that Mockito(more specifically [mockito-kotlin](https://github.com/nhaarman/mockito-kotlin)) - a library commonly used in Java and Android to provide mock test doubles and for behavior verification - cannot mock final classes.
+So why was the build failing? When the unit tests ran some of them would fail with an error stating that Mockito (mockitokotlin](https://github.com/nhaarman/mockito-kotlin)) - a library commonly used in Java and Android to provide mock test doubles and behavior verification - cannot mock final classes.
```

Comment:
The name of the library is `mockito-inline`, no reason to change it to a single word